src/routes/auth/docs.js

```diff
@@ -1,10 +1,3 @@
-import express from 'express';
```

Comment:
Why is there an `auth` folder inside the routes folder? Was it not agreed that each swagger definition should be a comment in the same file as the route definition?

Reply:
As a team we gound that the file will be too long. because in auth there will be sign up, login with email, social login, activate acct and reset password. So we decided to create a folder and put docs in their files

Reply:
I'd recommend you make use of the js file in the swagger config instead of using JSDOC comment for the swagger definition.

Reply:
That would mean that everytime a new route is created we go update the swagger config? Right?

Reply:
You can have each swagger definition inside the `src/swagger` directory then import each of them into the swagger config file.

Reply:
@adesege I didn't get this point correctly